queue/src/main/java/sjchat/queue/QueueChannel.java

```diff
@@ -0,0 +1,57 @@
+package sjchat.queue;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+public abstract class QueueChannel {
+
+  private Connection connection;
+  private Channel channel;
+
+  public abstract String toString();
+
+  public Channel getChannel() {
+    return channel;
+  }
+
+  public void initialize() throws QueueException {
+    String host = System.getenv("QUEUE_HOST");
+    host = (host == null) ? "localhost" : host;
+
+    try {
+      ConnectionFactory factory = new ConnectionFactory();
+      factory.setHost(host);
+
+      connection = factory.newConnection();
+      channel = connection.createChannel();
+
+    } catch (IOException exception) {
```

Comment:
@jvikstrom Reverted fix in commit 99417d8. Multi-catch statements is not supported by Docker's version of java.